.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: ❗️ Issue/Bug report
+about: Report issue or bug related to the project
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the issue:**
+<!-- A clear and concise description of what the bug is. If applicable, add screenshots to help explain your problem. -->
+
+**How to reproduce:**
+<!-- Steps to reproduce the behavior. -->
+
+**Expected behavior:**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
+ - Product Version: [e.g., IS 5.10.0, IS 5.9.0]
+ - OS: [e.g., Windows, Linux, Mac]
+ - Database: [e.g., MySQL, H2]
+ - Userstore: [e.g., LDAP, JDBC]
```

Comment:
Are these valid for the SDK?

Reply:
We can ask for the browser as well?

Reply:
Updated in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/19/commits/4de8fefaf82f63d07e8b131c972ec9039d08ed61